example-workflow.md

```diff
@@ -122,10 +124,10 @@ verification.
 
 *Example 1:* `alice.example`, in the Provider role, verifies the Relying Party
 (`bob.example`) that initiated the authentication process, and redirects Alice's
-browser to a pre-registered `redirect_uri` that `bob.example` provided during
-the previous Dynamic Registration step. So, Alice's browser gets redirected
-to `https://bob.example/auth-callback` (which is the URI Bob's POD uses as an OIDC
-`redirect_uri` endpoint).
+browser to a provided `redirect_uri` that `bob.example`. The `redirect_uri` MUST
```

Comment:
`to a provided `redirect_uri` that `bob.example`. The`
-> `to a `redirect_uri` provided by `bob.example`. The`
?

Reply:
Oops sorry about that.